packages/video_player_avplay/tizen/src/plus_player.cc

```diff
@@ -861,4 +864,7 @@ void PlusPlayer::OnCueOutContEvent(const char *cue_out_cont_data,
 
 void PlusPlayer::OnChangeSourceDone(bool ret, void *user_data) {}
 
-void PlusPlayer::OnStateChangedToPlaying(void *user_data) {}
+void PlusPlayer::OnStateChangedToPlaying(void *user_data) {
+  PlusPlayer *self = reinterpret_cast<PlusPlayer *>(user_data);
+  self->SendIsPlayingState(true);
```

Comment:
Just question: Is this callback called after buffering is complete?

media player changes state to true when calling player_start().
Of course, according to the description of player_start(), it says that the state will changes to PLAYER_STATE_PLAYING when called function, but I think it may work differently from plus player.
In fact, it may not be a problem. hahaha..

Reply:
The plusplayer has onStateChangedToPlaying callback, it will be invoked when player state was changed to playing,
we just need add this callback to listen playing state.
The media player doesn't have such callback, we need to send a playing state when calling play_start.